src/pages/index.tsx

```diff
@@ -18,27 +18,7 @@ const Welcome = () => {
 
   return (
     <>
-      <LangSwitcher />
       <Sidebar />
```

Comment:
you dont need this component on the welcome page

Reply:
I will fix this issue in RG-10